content/monitoring/deploy-grafana.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Deploy Grafana"
+date: 2018-10-14T20:54:13-04:00
+weight: 15
+draft: false
+---
+
+#### Download Grafana
+
+```
+curl -o grafana-values.yaml https://raw.githubusercontent.com/helm/charts/master/stable/grafana/values.yaml
+```
+
+Search for **# storageClassName: default**, uncomment and change the value to **"prometheus"**
```

Comment:
I'm wondering if we can come up with a better way to parameterize this file... maybe provide our own copy with pre-populated values when possible, to at least reduce the number of edits? We have a way to store/serve files that we are using in the onug branch.